codalab/lib/bundle_cli.py

```diff
@@ -1362,6 +1364,15 @@ def do_upload_command(self, args):
 
             # Canonicalize paths (e.g., removing trailing /)
             sources = [path_util.normalize(path) for path in args.path]
+            # Calculate size of sources
+            total_bundle_size = sum([get_path_size(source) for source in sources])
+            user = client.fetch('users', client.fetch('user')['user_name'])
+            disk_left = user['disk_quota'] - user['disk_used']
+            if disk_left - total_bundle_size <= 0:
+                raise DiskQuotaExceededError(
+                    'Attempted to upload bundle of size %d with only %d remaining.'
```

Comment:
```suggestion
                    'Attempted to upload bundle of size %d with only %d remaining in user\'s disk quota.'
```

Reply:
@adiprerepa can you make this change in another PR?

Reply:
#3409